string pos1
string pos2
string team
int32  aruco_name
---
